### DISCUSSION TITLE
支持 Dai 2.0 跨链

### DISCUSSION THREAD

**Discussion by charliex2** [FIBOSIO/fibos#21] [Ideas]
目前 Dai 1.0 已经不流通，流通的均是 Dai 2.0。但是 fibos 跨链目前还仅支持 Dai 1.0，希望能支持 Dai 2.0.
目前 fo wallet 对 dai 版本没有任何提示，目前容易跨 Dai 2.0 进来，然后没到账。

**Comment by luoyhang003** [comment]
对的，这个需要尽快支持一下了
